Api/namedPipe/namedPipe.h

```diff
@@ -0,0 +1,17 @@
+// From http://stackoverflow.com/questions/2784500/how-to-send-a-simple-string-between-two-programs-using-pipes
+#include <fcntl.h>
+#include <sys/stat.h>
+#include <sys/types.h>
+#include <sys/time.h>
+#include <unistd.h>
+
+
+#define BLOCK 1024 
+
+int openNamedPipe(char * something);
+
+void writeNamedPipe(int fd, void * data, int size);
```

Comment:
Those should be "private" functions( I don't know how to do it in C. Java <3.), there should be one function (the one on the user's side) that call this one.